### PR TITLE
OWDistances: warn about imputing missing values

### PR DESCRIPTION
**Pull request by kernc**

### CODE REVIEW

File: Orange/widgets/unsupervised/owdistances.py [hunk 1]
```diff
@@ -121,6 +123,8 @@ def commit(self):
             if numpy.product(shape) == 0:
                 self.Error.empty_data(shape)
             else:
+                if bn.anynan(data.X):
```
Comment (lanzagar) [inline Orange/widgets/unsupervised/owdistances.py]: Can this be moved closer to the actual check for missing data and imputation. Reusing the same conditional check if possible (currently appears to be coupled with discretization, so might need some exercise).

**Comment by ajdapretnar** [comment]
Isn't this supposed to show warning when imputing missing values in Distances widget? Why can't I see this message?

**Comment by lanzagar** [comment]
Fixed in #1551

**Comment by kernc** [comment]
I don't see how this is _replaced_ by #1551. The point of warning of imputation is that you can get different results depending on imputation axis.

**Comment by kernc** [comment]
Nevermind.

**Pull request closed without merging**